hopsworks-common/src/main/java/io/hops/hopsworks/common/util/Settings.java

```diff
@@ -1382,6 +1392,14 @@ public String getHadoopClasspathGlob() throws IOException {
       + File.separator;
   public static final String PROJECT_STAGING_DIR = "Resources";
 
+  private static String DEFAULT_RESERVED_PROJECT_NAMES = "python27,python36,hops-system,airflow,ndbinfo,mysql";
```

Comment:
@kouzant why the default list does not match [https://github.com/logicalclocks/hopsworks-chef/pull/356/files#diff-25e5d4a4446ae12a0d6f1162b6160375R128](https://github.com/logicalclocks/hopsworks-chef/pull/356/files#diff-25e5d4a4446ae12a0d6f1162b6160375R128) ?

Reply:
I updated the default reserved project names to match what it was before. But we should avoid duplicating them. I suppose the purpose of the default value is Hopsworks not crash badly if we have wrongly forgotten to set something in the database.